packages/@navikt/side-menu/src/MenuLinkStyles.less

```diff
@@ -70,4 +70,35 @@
     &__icon {
         margin-left: 1rem;
     }
+
+    &__arrow-theme {
+        .menu-link__button {
+            &--active {
```

Comment:
Kan vi heller style dette som modifiers på den stylingen som vi allerede har for `menuLink`? Per nå ser det ut til at `__arrow-theme` er stylet som et element under `.menu-link`.

Tenker f.eks. noe ala dette
```
.menu-link {
  &--withArrows {
    &__button {
     // styling here  
    }
  } 
}
```

Reply:
Forøvrig også en kompliserende greie som jeg tenker gjør at denne komponenten bør splittes opp mer etterhvert.

Reply:
Enig